frontend/src/pages/article/components/article-form/article-form.tsx

```diff
@@ -32,9 +40,24 @@ const ArticleForm: React.FC = () => {
               : null,
         };
 
-        void dispatch(articlesActions.createArticle(updatedPayload));
+        generatedPrompt
+          ? void dispatch(
+              articlesActions.createArticle({
+                articlePayload: updatedPayload,
+                generatedPrompt: {
+                  type: PromptType.MANUAL,
+                  ...generatedPrompt,
+                },
+              }),
+            )
+          : void dispatch(
+              articlesActions.createArticle({
+                articlePayload: updatedPayload,
+                generatedPrompt: null,
+              }),
+            );
```

Comment:
Or even move getGeneratedPrompt to helper fn to make it more readable
```suggestion
        void dispatch(
              articlesActions.createArticle({
                articlePayload: updatedPayload,
                generatedPrompt: generatedPrompt
                  ? {
                    type: PromptType.MANUAL,
                    ...generatedPrompt,
                  }
                  : null,
              }),
            );
```